test/integ_tests/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs_mocks.py

```diff
@@ -57,4 +97,4 @@ def post_run(tb):
         os.remove("results.json")
         """
     )
-
+    mock_cloudpickle.stop()
```

Comment:
You call start() in the injected code but stop is not injected. Is that intended? If not, you can just move it into the string above.

Reply:
Ah missed how that worked, updated